Modules/Core/SpatialObjects/include/itkSpatialObject.h

```diff
@@ -342,9 +352,14 @@ class ITK_TEMPLATE_EXPORT SpatialObject : public DataObject
   /** World space equivalent to IsEvaluableAtInObjectSpace
    * \note This member function assumes that the internal `ObjectToWorldTransformInverse` transform is up-to-date. This
    * transform may be updated explicitly by calling `GetObjectToWorldTransformInverse()`, `Update()`, or
-   * `SetObjectToWorldTransform(transform)` */
-  virtual bool
-  IsEvaluableAtInWorldSpace(const PointType & point, unsigned int depth = 0, const std::string & name = "") const;
+   * `SetObjectToWorldTransform(transform)`
+   * \note This member function is not meant to be overridden. In the future, it may not be declared `virtual` anymore.
```

Comment:
BTW, the statement (`This member function is not meant to be overridden.`) seems to conflict with the explanation above (lines 47-52):

```
 * <...> To
 * implement your own spatial object, you need to derive from the
 * following class, which requires overriding just a few virtual functions.
 * Examples of such functions are ValueAtInObjectSpace(),
 * IsInsideInObjectSpace(), and ComputeMyBoundingBox(), each of which has a
 * meaning specific to each particular object type.
```